packages/runtime/container-runtime/package.json

```diff
@@ -230,6 +230,10 @@
 		"typescript": "~5.4.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"RemovedVariable_AllowInactiveRequestHeaderKey": {
```

Comment:
Don't you need to leave it for now, and merely deprecate it?

Reply:
Never mind it's `@internal`.  @tylerbutler do we have plans to stop doing type tests for internal stuff? Please?

Reply:
Yes, it was supposed to be done this sprint but I didn't get to it. Pushed to next sprint.